src/css/styles.scss

```diff
@@ -53,82 +56,82 @@ header {
   grid-area: header;
   padding: 0% 0%;
   width: 100%;
-}
 
-span {
-  color: #C4EB67;
-  font-weight: 400;
-}
+  span {
+    color: $bright-border-color;
+    font-weight: 400;
+  }
 
-.header-apple-icon {
-  height: 40px;
-  margin-left: 5px;
-  width: 35px;
-}
+  .header-apple-icon {
+    height: 40px;
+    margin-left: 5px;
+    width: 35px;
+  }
+};
 
 #search {
   border-radius: 15px 15px 15px 15px;
-  border: 3px outset #C4EB67;
-  box-shadow: 0px 0px 7px 0px #475d74;
+  border: 3px outset $bright-border-color;
+  box-shadow: 0px 0px 7px 0px $box-shadow-background-color;
   display: flex;
   flex-direction: row;
-  font-family: "Quicksand", sans-serif;
+  font-family: $default-body-font-family, sans-serif;
   justify-content: center;
   margin: 1.2% 0.5% 1.2% 0%;
   width: 100%;
-}
-
-#search-input {
-  background-color: #b6dcca;
-  background: -webkit-gradient( linear,left top, left bottom, color-stop(0, rgb(255, 255, 255)), color-stop(1, rgb(225,225,225)));
-  border-radius: 10px 0px 0px 10px;
-  border: none;
-  box-shadow: inset -3px -1px 20px 1px #5c7894;
-  color: #5B7894;
-  font-family: "Quicksand", sans-serif;
-  font-size: 42px;
-  height: 100%;
-  outline: none;
-  padding: 0% 2% 0% 0%;
-  text-align: right;
-  transition: .8s ease;
-  width: 100%;
-}
-
-#search-input:hover {
-  box-shadow: inset 0px 0px 0px 0px white;
-  transition: .8s ease;
-}
-
-#search-input:focus {
-  box-shadow: inset 0px 0px 0px 0px white;
-}
-
-#search-input::placeholder {
-  color: #5B7894;
-  font-size: 20px;
-  text-align: right;
-  text-shadow: none;
-}
 
-#search-input:focus::placeholder {
-  font-size: 0px;
-  transition: .8s ease;
-}
-
-.search-label {
-  display: none;
-}
-
-.search-btn {
-  border-left: 3px outset #C4EB67;
-  border-radius: 0px 15px 15px 0px;
-  padding: 1%;
-}
+  #search-input {
+    background-color: #b6dcca;
+    background: -webkit-gradient( linear,left top, left bottom, color-stop(0, rgb(255, 255, 255)), color-stop(1, rgb(225,225,225)));
+    border-radius: 10px 0px 0px 10px;
+    border: none;
+    box-shadow: inset -3px -1px 20px 1px #5c7894;
+    color: $blue-background-color;
+    font-family: $default-body-font-family, sans-serif;
+    font-size: 42px;
+    height: 100%;
+    outline: none;
+    padding: 0% 2% 0% 0%;
+    text-align: right;
+    transition: .8s ease;
+    width: 100%;
+  }
+
+  #search-input:hover {
```

Comment:
Did we try nesting these search input elements?

Reply:
I tried that by nesting the search ingredients, title, input, results, icon in the search ingredients button but that moved the search bar and messed up the display. I can try and see which elements I can nest.